packages/web-components/src/components/va-alert/va-alert.tsx

```diff
@@ -49,7 +57,7 @@ export class VaAlert {
   /**
    * Aria-label text for the close button.
    */
-  @Prop() closeBtnAriaLabel: string = 'Close notification';
+  @Prop() closeBtnAriaLabel: string = i18next.t('close-notification');
```

Comment:
I don't think this is something that we want right now unless things have changed since [this comment](https://github.com/department-of-veterans-affairs/component-library/issues/343#issuecomment-1097210319).

> They are just going to override the value for va-loading-indicator and va-alert. I let them know which props they would need to pass in and made sure they understood that if they don't the text will be in English.

cc @k80bowman for awareness.

Reply:
No, let's not do this one right now. We don't have the translated text for it.

Reply:
@bkjohnson & @k80bowman closing this PR for now then.